src/uct/api/tl.h

```diff
@@ -344,6 +346,7 @@ typedef struct uct_iface_ops {
     uct_ep_connect_func_t               ep_connect;
     uct_ep_disconnect_func_t            ep_disconnect;
     uct_cm_ep_conn_notify_func_t        cm_ep_conn_notify;
+    uct_ep_query_func_t                 ep_query;
```

Comment:
pls add this to internal_ops to avoid UCT ABI break

Reply:
@yosefe Do you mean move to? Remove the change here, and add to internal_ops?

Reply:
@yosefe Done